samples/createPushSubscriptionNoWrapper.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is a generated sample, using the typeless sample bot. Please
+// look for the source TypeScript sample (.ts) for modifications.
+'use strict';
+
+/**
+ * This application demonstrates how to perform basic operations on
+ * subscriptions with the Google Cloud Pub/Sub API.
+ *
+ * For more information, see the README.md under /pubsub and the documentation
+ * at https://cloud.google.com/pubsub/docs.
+ */
+
+// sample-metadata:
+//   title: Create Push Subscription With No Wrapper
+//   description: Creates a new push subscription, but disables metadata wrapping.
```

Comment:
writeMetadata seems to be set to `true` in the sample. Is this comment taken from another sample? If so, maybe we should clarify using the language from the proto https://github.com/googleapis/googleapis/blob/87e73054cd98b8d4a3f516066c6530213362bef4/google/pubsub/v1/pubsub.proto#L966 and https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub.proto#L914-L917

Reply:
Yeah it's the wrapping it's disabling, not the writing, but I think this might be leftover language from the pre-"disable wrapping" change.

Reply:
I like the proto language better. I've updated it.